# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1392949965 18000
#      Thu Feb 20 21:32:45 2014 -0500
# Branch gui-release
# Node ID b75adad91fc62e6356294b4e4b281d6ad3643fe4
# Parent  554be77a60fb9a27bce21ff3e38547126d843d18
Add missing source files to libgui/graphics/module.mk

* libgui/graphics/module.mk (noinst_HEADERS): Add ToolBarButton.h.
(graphics_libgui_graphics_la_SOURCES): Add ToolBarButton.cc.

diff --git a/libgui/graphics/module.mk b/libgui/graphics/module.mk
--- a/libgui/graphics/module.mk
+++ b/libgui/graphics/module.mk
@@ -58,16 +58,17 @@ noinst_HEADERS += \
   graphics/PushTool.h \
   graphics/RadioButtonControl.h \
   graphics/SliderControl.h \
   graphics/TextControl.h \
   graphics/TextEdit.h \
   graphics/ToggleButtonControl.h \
   graphics/ToggleTool.h \
   graphics/ToolBar.h \
+  graphics/ToolBarButton.h \
   graphics/Utils.h \
   graphics/gl-select.h
 
 graphics_libgui_graphics_la_SOURCES = \
   graphics/__init_qt__.cc \
   graphics/Backend.cc \
   graphics/BaseControl.cc \
   graphics/ButtonControl.cc \
@@ -93,16 +94,17 @@ graphics_libgui_graphics_la_SOURCES = \
   graphics/PushTool.cc \
   graphics/RadioButtonControl.cc \
   graphics/SliderControl.cc \
   graphics/TextControl.cc \
   graphics/TextEdit.cc \
   graphics/ToggleButtonControl.cc \
   graphics/ToggleTool.cc \
   graphics/ToolBar.cc \
+  graphics/ToolBarButton.cc \
   graphics/Utils.cc \
   graphics/gl-select.cc
 
 nodist_graphics_libgui_graphics_la_SOURCES = $(octave_gui_graphics_MOC) $(octave_gui_graphics_RC)
 
 graphics_libgui_graphics_la_CPPFLAGS = \
   $(AM_CPPFLAGS) \
   $(FT2_CFLAGS) \
